architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,86 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization determines whether an identity is allowed to perform any action.
+Note that authorization is a separate step from
+link:authentication.html[authentication], which is more about determining the
+identity of who is taking the action.
+
+Authorization is managed using:
+
+[horizontal]
+Rules:: Describe sets of permitted verbs on a set of resource types.
+Roles:: Collection of rules.
+Bindings:: Associate users and/or groups with a role.
+
+Users and groups may be bound to multiple roles at the same time.
+
+Several factors are combined to make the decision when authorization is
+evaluated:
```

Comment:
Who makes the decision? Who is evaluating the authorization? I think this should be rewritten to clarify a bit more, it's not very clear from the customer's perspective in my opinion.